docs/QUICKSTART.md

```
@@ -0,0 +1,298 @@
+git clone https://github.com/LucasSantana-Dev/mcp-gateway.git && \
+cd mcp-gateway && \
+./scripts/mcp init && \
+./scripts/mcp start && \
+./scripts/mcp ide setup
+```
+
+---
+
+## 🔧 Essential Commands
+
+### Gateway Management
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)